content/blog/what-is-nextjs.md

```diff
@@ -1,4 +1,5 @@
 ---
+date: '2020-11-07'
```

Comment:
Had to reselect the date from the date picker to add it back. I had no clue, something was wrong as a date was showing up in the sidebar.